templates/forms/license-activation.php

```diff
@@ -239,6 +255,110 @@ class="fs-available-license-key"
             totalSites           = <?php echo count( $sites_details ) ?>,
             singleBlogID         = null;
 
+        var
+            previousLicenseKey          = null,
+            licenseUserDataByLicenseKey = {},
```

Comment:
@fajardoleo as discussed, no need to store the returned data for a collection of licenses, since the chance a user will type one key, then another key, and then the 1st key again is slim. Instead, just store the latest license's user data result from the API.